gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,173 @@
+ */
+import React from 'react';
+import { NxCode, NxP, NxTable, NxTile, NxH2, NxTextLink } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxToastExample from './NxToastExample';
+import NxToastVariationsExample from './NxToastVariationsExample';
+import NxToastMultipleExample from './NxToastMultipleExample';
+import CodeExample from '../../CodeExample';
+
+const nxToastExampleCode = require('./NxToastExample?raw'),
+    nxToastVariationsExampleCode = require('./NxToastVariationsExample?raw'),
+    nxToastMultipleExampleCode = require('./NxToastMultipleExample?raw'),
+    nxGlobalHeaderToastExampleCode = require('../../styles/NxGlobalHeader/NxGlobalHeaderToastExample?raw');
+
+const NxToastPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Toasts are rendered on the top right corner of the screen. RSC provides 4 different variations of
+        Toasts: success, warning, info, and error.
+      </NxP>
+      <NxP>
+        Unlike all of the other React components provided by the RSC, the RSC toast component has a
+        slightly different API. While other components can simply be used directly by using the component
+        names, for example, using <NxCode>{'<NxList>'}</NxCode> to render a list or
+        using <NxCode>{'<NxAlert>'}</NxCode> to render an alert, the pattern for rendering
+        toasts is slightly different. Instead of using component names, the following functions are used
+        to display the different kinds of toasts:
+      </NxP>
+      <NxP>
+        <NxCode>showErrorToast()</NxCode> to render an error toast.
+      </NxP>
+      <NxP>
+        <NxCode>showSuccessToast()</NxCode> to render a success toast.
+      </NxP>
+      <NxP>
+        <NxCode>showWarningToast()</NxCode> to render a warning toast.
+      </NxP>
+      <NxP>
+        <NxCode>showInfoToast()</NxCode> to render an info toast.
+      </NxP>
+      <NxP>
+        Toasts generally fall under a 'global and static'
+        context, in a sense that throughout an application, toasts are rendered within the same
+        container on the top-right corner of the screen. Keeping this in mind, the most effective way of rendering
+        the toasts is to wrap the parent root React component with <NxCode>NxToastProvider</NxCode>.
+        RSC also provides a convenience React hook <NxCode>useToast</NxCode> that contains all of the functions
+        for rendering toasts. The <NxCode>useToast</NxCode> hook is used as follows:{' '}
+      </NxP>
+      <NxP>
+        <NxCode>
+          {'const { showErrorToast, showSuccessToast, showWarningToast, showInfoToast } = useToast();'}
+        </NxCode>
+      </NxP>
+      <NxP>
+        When these functions to display the various kinds of toasts, i.e. <NxCode>showErrorToast()</NxCode>,
+        {' '}<NxCode>showSuccessToast()</NxCode>, <NxCode>showWarningToast()</NxCode>,
```

Comment:
Reiterating the four function names here seems a little bit redundant and cluttered to me

Reply:
Removed the function names here: [91c0683](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/91c068321f25b891bf2f411e586fa7a6101eeef3)